content/en/docs/installation/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Installation"
+linkTitle: "Installation"
+weight: 2
+description: |
+  Installing Spinnaker
+aliases:
+  - /install_guide/install/
+  - /install-guide/gettings_started/
+---
+
+Below is a summary of the different installation methods:
+
+| Method  | Environment | Description  | Pros |
+|---|---|--|--|
+| [Spinnaker Operator](./operator) | Kubernetes | Kubernetes operator that turns Spinnaker's configuration declarative | GitOps friendly, Production use | 
+| [Armory Halyard](./armory-halyard)  | Kubernetes | Versatile command line interface to configure and deploy Spinnaker | Quick Setup |
+| [Minnaker](./minnaker) | MacOS, Linux, Windows | Spin up a whole environment on top of Rancher to deploy Spinnaker | This is ideal if you do not have a Kubernetes cluster available |
+
+
+All the methods above share similar configurations and you can change between them.
```

Comment:
Do we have something to guide users on how to move between configuration formats? Feels odd to call this out here without giving users a lifeline hah

Reply:
Yes for operator. No for the other ones. I see what you're getting at but was hoping to keep that for a future iteration.